src/cemerick/friend/util.clj

```diff
@@ -12,9 +12,14 @@
 ; this was never _really_ part of the API, was implemented (badly) before req/request-url was available
 (def ^:deprecated original-url req/request-url)
```

Comment:
Now this isn't used anywhere in the codebase but I guess it's better to keep it around just in case external clients are using it?